elytron/src/main/java/org/wildfly/extension/elytron/_private/ElytronSubsystemMessages.java

```diff
@@ -557,4 +557,6 @@
     @Message(id = 1060, value = "Fileless KeyStore needs to have a defined type.")
     OperationFailedException filelessKeyStoreMissingType();
 
+    @Message(id = 1061, value = "Value of host context map contains invalid hostname.")
```

Comment:
This message should include the invalid value.

Reply:
@bstansberry Thank you. Fixed.